elements/search/simple/nuxeo-simple-search-results.html

```diff
@@ -0,0 +1,114 @@
+<!--
+`nuxeo-simple-search-results`
+@group Nuxeo UI
+@element nuxeo-simple-search-results
+-->
+<dom-module id="nuxeo-simple-search-results">
+  <template>
+    <style include="nuxeo-styles">
+      nuxeo-data-table,
+      nuxeo-data-grid,
+      nuxeo-data-list,
+      nuxeo-justified-grid {
+        display: block;
+        position: relative;
+        min-height: calc(100vh - 130px - var(--nuxeo-app-top));
+      }
+
+      nuxeo-justified-grid {
+        height: calc(100vh - 10em);
+      }</style
+    ><nuxeo-results
```

Comment:
Not sure if it's the formatter doing this but it looks a bit funky.
```suggestion
      }
    </style>
    <nuxeo-results
```

Reply:
AFAIK these files are only meant to test the solution suggested

Reply:
Yes, please ignore the second commit. Not going to merge that one.